### ISSUE TITLE
HELP !!!!!!!!!!! I

### ISSUE THREAD

**Issue by CptJoker** [WeebSec/PhishX#51]
it just comes back to main menu after selecting template and it says attack has started later come backs to same menu please help 

Thanks in Advance :)

**Comment by Gmm33** [comment]
Me too please help

**Comment by Gmm33** [comment]
Even when i change the code in phishx.py

**Comment by LulzsecHackz** [comment]
Something is wrong in the code.... The code runs with Geckodriver, some OS' Doesn't support it or just crash..... U need to change the code..... Instead of firefox driver, use Chrome Driver... If u need more help ill send u the modified code.

**Comment by CharlesTheGreat77** [comment]
@LulzsecHackz How would I go about changing the Driver to Chrome rather than Firefox

**Comment by sameer2012** [comment]
Hiiii...@LulzsecHackz.   Please send me modified code... plz

**Comment by Kingkhan12398** [comment]
@LulzsecHackz I need your help. please give me modified code. i have same problem" when attack start then it goes back to main menu

**Comment by Parshuram2** [comment]
> @LulzsecHackz I need your help. please give me modified code. i have same problem" when attack start then it goes back to main menu

There is no need for modified code.
The errors are not in the script per se but depends on your system.
First you download Firefox from firefox website. Replace the firefox folder in the 'base' folder'. The version you download will be 66.6
Then google gecko driver download. Download geckodriver version 24. Copy it in the 'base' folder to replace gecko driver 0.18.0

Type "locate selenium" in your terminal. See which version it is. If it is 3.5 or something like that type
pip uninstall selenium.

Then type pip install -U selenium. This will download selenium version 3.14. Now, you have the ingredients right.

If you have problems. Write here again.

**Comment by Kingkhan12398** [comment]
It shows firefox version 66.0.2 but not 66.6 and also how to install and replace Firefox and geckodriver? Also tell me how to check your geckodriver and Firefox current version? Thanks

**Comment by Kingkhan12398** [comment]
I am using smptp2go for spoofed email. Is it good?

**Comment by Kingkhan12398** [comment]
[](url)
![IMG_20190410_034741](https://user-images.githubusercontent.com/49414117/55840045-74154c00-5b43-11e9-9048-6bfbefe03895.jpg)it goes to main menu from here

**Comment by Parshuram2** [comment]
> ![IMG_20190410_034741](https://user-images.githubusercontent.com/49414117/55840045-74154c00-5b43-11e9-9048-6bfbefe03895.jpg)it goes to main menu from here

Version 66 is good enough. smtp2go  won't allow sending mails to gmail.yahoo etc (As far as I can remember). 

Download Firefox from Firefox website. You will get a zip file. Unzip it. You will get a folder called "firefox". Simply copy it in the "base" folder in PhisX folder. 
Go here https://github.com/mozilla/geckodriver/releases . Donwload version 24. Unzip it. copy and paste in the "base" folder to replace the existing geckodriver. Then copy paste the geckodriver v 24 in /usr/local/bin folder as well.

selenium ..read above how to update.

**Comment by Parshuram2** [comment]
> ![IMG_20190410_034741](https://user-images.githubusercontent.com/49414117/55840045-74154c00-5b43-11e9-9048-6bfbefe03895.jpg)it goes to main menu from here

To find out what is the exact error do this. 
Type "sudo python3 -m pdb PhishX.py
you will get a pdb prompt.
type "break PhishX.py:311"
type "enable 1"
type "b" and press enter to see the breakpoint.
type "cont" press enter
The program will run normally.
Select "Twitter"
After some time you will see the pdb prompt again.
Simply type "n" and press enter. This will execute the program line by line. Wait for at least 10-11 seconds in case you don't see anything happening after typing "n" and pressing "enter".
If nothinh happens after the line "browser.get("http://localhost:4040/status") ..do the above. That is ..change the versions of selenium,firefox,and geckodriver. 

If you see a selenium error something like "selenium.commonm.exception"..blah blah...copy it and paste it here.

**Comment by Kingkhan12398** [comment]
I setup sendgrid. Configured properly but it says email sent successfully but it doesn't goes to receiver email. Also when I tried to find out what exact problem is then it runs good till PhishX run normally but from then pdb prompt never come again after waiting for so long. So I didn't able to configure out what actuall problem is

**Comment by Kingkhan12398** [comment]
I setup smptp2go which was sending mails without any problem. I also added domain in smptp2go for better performance and it is sending mails to Yahoo and Gmail without anyone problem. But sendgrid isn't working for me I think

**Comment by Kingkhan12398** [comment]
[
![IMG_20190410_205302](https://user-images.githubusercontent.com/49414117/55894125-01ec4800-5bd3-11e9-9af4-c84a1d04ece1.jpg)
![IMG_20190410_205226](https://user-images.githubusercontent.com/49414117/55894126-0284de80-5bd3-11e9-8306-3ed84cc1fde6.jpg)
![IMG_20190410_034741](https://user-images.githubusercontent.com/49414117/55894127-0284de80-5bd3-11e9-9793-454e658195ee.jpg).  Nothing happened after that
![IMG_20190410_205357](https://user-images.githubusercontent.com/49414117/55894128-031d7500-5bd3-11e9-9404-327ba104d5d4.jpg)
![IMG_20190410_205312](https://user-images.githubusercontent.com/49414117/55894130-031d7500-5bd3-11e9-871d-a531150659d3.jpg)
![IMG_20190410_205412](https://user-images.githubusercontent.com/49414117/55894131-03b60b80-5bd3-11e9-94e5-febd52e5f5ec.jpg)
![IMG_20190410_205443](https://user-images.githubusercontent.com/49414117/55894134-03b60b80-5bd3-11e9-8d81-e83ca3820e08.jpg)
![IMG_20190410_205436](https://user-images.githubusercontent.com/49414117/55894138-03b60b80-5bd3-11e9-8d8e-7b61a743cf8d.jpg)
![IMG_20190410_205428](https://user-images.githubusercontent.com/49414117/55894141-044ea200-5bd3-11e9-83f5-0548f10291ba.jpg)








](url)

**Comment by Parshuram2** [comment]
If it says "Blank or Comment" ..just change the line number to something else.  like break PhishX.py:313. 
type "leafpad PhishX.py" .....select options..enable line numbers...see what the line number is which is not a comment or blank. Use Twitter. Then it will show weather smtp is a problem or not. So, forget about changing smtp for now.

**Comment by Kingkhan12398** [comment]
i chnaged line number. it shows breakpoint now but still pdb prompt not appear after normal running of PhishX
    On Wednesday, April 10, 2019, 12:06:47 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 
If it says "Blank or Comment" ..just change the line number to something else. like break PhishX.py:313.
type "leafpad PhishX.py" .....select options..enable line numbers...see what the line number is which is not a comment or blank. Use Twitter. Then it will show weather smtp is a problem or not. So, forget about changing smtp for now.

**Comment by Parshuram2** [comment]
> i chnaged line number. it shows breakpoint now but still pdb prompt not appear after normal running of PhishX On Wednesday, April 10, 2019, 12:06:47 PM EDT, Parshuram2 <notifications@github.com> wrote: If it says "Blank or Comment" ..just change the line number to something else. like break PhishX.py:313. type "leafpad PhishX.py" .....select options..enable line numbers...see what the line number is which is not a comment or blank. Use Twitter. Then it will show weather smtp is a problem or not. So, forget about changing smtp for now. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

did you type "enable 1" after break PhishX.py:31x?? ...do that and then type "b" and press enter to verify the breakpoint.

**Comment by Parshuram2** [comment]
> i chnaged line number. it shows breakpoint now but still pdb prompt not appear after normal running of PhishX On Wednesday, April 10, 2019, 12:06:47 PM EDT, Parshuram2 <notifications@github.com> wrote: If it says "Blank or Comment" ..just change the line number to something else. like break PhishX.py:313. type "leafpad PhishX.py" .....select options..enable line numbers...see what the line number is which is not a comment or blank. Use Twitter. Then it will show weather smtp is a problem or not. So, forget about changing smtp for now. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

also...type sudo before the command. python3 -m pdb PhishX.py..so type "sudo python3 -m pdb PhishX.py

**Comment by Kingkhan12398** [comment]
did everything what you said
    On Wednesday, April 10, 2019, 12:16:44 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 

i chnaged line number. it shows breakpoint now but still pdb prompt not appear after normal running of PhishX On Wednesday, April 10, 2019, 12:06:47 PM EDT, Parshuram2 notifications@github.com wrote: If it says "Blank or Comment" ..just change the line number to something else. like break PhishX.py:313. type "leafpad PhishX.py" .....select options..enable line numbers...see what the line number is which is not a comment or blank. Use Twitter. Then it will show weather smtp is a problem or not. So, forget about changing smtp for now. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.


did you type "enable 1" after break PhishX.py:31x?? ...do that and then type "b" and press enter to verify the breakpoint.

**Comment by Parshuram2** [comment]
what happens when you type "enable 1"...Tell me the error.

**Comment by Kingkhan12398** [comment]
it shows it enabled. and when i type b it shows breakpoint
    On Wednesday, April 10, 2019, 12:21:35 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 
what happens when you type "enable 1"...Tell me the error.

**Comment by Parshuram2** [comment]
so type "cont" press enter and run the program normally. till it comes back to the pdb prompt.

**Comment by Kingkhan12398** [comment]
run it normally but i wait for like 10 mins
    On Wednesday, April 10, 2019, 12:24:43 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 
so type "cont" press enter and run the program normally. till it comes back to the pdb prompt.

**Comment by Parshuram2** [comment]
no. If you have put break  at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315.

**Comment by Kingkhan12398** [comment]
haha but i dont have any twitter details. send me some fake ones    On Wednesday, April 10, 2019, 12:28:28 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 
no. If you have put break at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315.

**Comment by Kingkhan12398** [comment]
i put line number 313
    On Wednesday, April 10, 2019, 12:28:28 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 
no. If you have put break at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315.

**Comment by Parshuram2** [comment]
> haha but i dont have any twitter details. send me some fake ones On Wednesday, April 10, 2019, 12:28:28 PM EDT, Parshuram2 <notifications@github.com> wrote: no. If you have put break at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

Twitter details?..Put anything ..like "sdsdF" or some shit for username but put your own email address in email option.

**Comment by Kingkhan12398** [comment]
when i put twitter details then attack started abd some other window like php opened. some preocess done there i didnt able to read what it doing. it came for 3 4 secs. then again normal phishx menu is opened. but no pdb prompt
    On Wednesday, April 10, 2019, 12:38:02 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 

haha but i dont have any twitter details. send me some fake ones On Wednesday, April 10, 2019, 12:28:28 PM EDT, Parshuram2 notifications@github.com wrote: no. If you have put break at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.


Twitter details?..Put anything ..like "sdsdF" or some shit for username but put your own email address in email option.

**Comment by Kingkhan12398** [comment]
On Wednesday, April 10, 2019, 12:40:56 PM EDT, trunbooking@yahoo.com <trunbooking@yahoo.com> wrote:  
 
  when i put twitter details then attack started abd some other window like php opened. some preocess done there i didnt able to read what it doing. it came for 3 4 secs. then again normal phishx menu is opened. but no pdb prompt
    On Wednesday, April 10, 2019, 12:38:02 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 

haha but i dont have any twitter details. send me some fake ones On Wednesday, April 10, 2019, 12:28:28 PM EDT, Parshuram2 notifications@github.com wrote: no. If you have put break at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.


Twitter details?..Put anything ..like "sdsdF" or some shit for username but put your own email address in email option.

**Comment by Parshuram2** [comment]
> when i put twitter details then attack started abd some other window like php opened. some preocess done there i didnt able to read what it doing. it came for 3 4 secs. then again normal phishx menu is opened. but no pdb prompt On Wednesday, April 10, 2019, 12:38:02 PM EDT, Parshuram2 <notifications@github.com> wrote: haha but i dont have any twitter details. send me some fake ones On Wednesday, April 10, 2019, 12:28:28 PM EDT, Parshuram2 notifications@github.com wrote: no. If you have put break at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread. Twitter details?..Put anything ..like "sdsdF" or some shit for username but put your own email address in email option. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

did you run the command with sudo? sudo python3 -m pdb PhishX.py?

**Comment by Kingkhan12398** [comment]
yes i did run with sudo. I did everything exactly how you told me

    On Wednesday, April 10, 2019, 12:44:39 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 

when i put twitter details then attack started abd some other window like php opened. some preocess done there i didnt able to read what it doing. it came for 3 4 secs. then again normal phishx menu is opened. but no pdb prompt On Wednesday, April 10, 2019, 12:38:02 PM EDT, Parshuram2 notifications@github.com wrote: haha but i dont have any twitter details. send me some fake ones On Wednesday, April 10, 2019, 12:28:28 PM EDT, Parshuram2 notifications@github.com wrote: no. If you have put break at line 313 or something like that...type "cont" press enter...select Twitter attack...enter Twitter details...then it should come back to pdb prompt in like 10 -15 seconds. Which line number did you put a break at? it should be something close to 310-315. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread. Twitter details?..Put anything ..like "sdsdF" or some shit for username but put your own email address in email option. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.


did you run the command with sudo? sudo python3 -m pdb PhishX.py?

**Comment by Parshuram2** [comment]
Type "Leafpad PhishX.py"...then look for this code in Twitter part of the code.

# `PILE O SHIT BELOW...

	time.sleep(3)
	display = Display(visible=0, size=(800, 600))
	display.start()
	browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')
	browser.get("http://localhost:4040/status")
	time.sleep(5)
	url_f=str(browser.find_element_by_css_selector("li.list-unstyled:nth-child(1) > div:nth-child(2) > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2)").text)
	browser.close()
	phishin_url = str(url_f)

Try putting the break on diffrent line numbers. But the line number must be something BEFORE the "browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')" command.  Try a few line numbers. Once you get the pdb prompt . type "n" and then enter..till you see an error or the program gets stuck.

**Comment by Kingkhan12398** [comment]
tried all lines between 310 and 315 but not happening
    On Wednesday, April 10, 2019, 12:52:47 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 
Type "Leafpad PhishX.py"...then look for this code in Twitter part of the code.

`PILE O SHIT BELOW...
time.sleep(3)
display = Display(visible=0, size=(800, 600))
display.start()
browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')
browser.get("http://localhost:4040/status")
time.sleep(5)
url_f=str(browser.find_element_by_css_selector("li.list-unstyled:nth-child(1) > div:nth-child(2) > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2)").text)
browser.close()
phishin_url = str(url_f)

Try putting the break on diffrent line numbers. But the line number must be something BEFORE the "browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')" command. Try a few line numbers. Once you get the pdb prompt . type "n" and then enter..till you see an error or the program gets stuck.

**Comment by Parshuram2** [comment]
> tried all lines between 310 and 315 but not happening On Wednesday, April 10, 2019, 12:52:47 PM EDT, Parshuram2 <notifications@github.com> wrote: Type "Leafpad PhishX.py"...then look for this code in Twitter part of the code. `PILE O SHIT BELOW... time.sleep(3) display = Display(visible=0, size=(800, 600)) display.start() browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver') browser.get("http://localhost:4040/status") time.sleep(5) url_f=str(browser.find_element_by_css_selector("li.list-unstyled:nth-child(1) > div:nth-child(2) > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2)").text) browser.close() phishin_url = str(url_f) Try putting the break on diffrent line numbers. But the line number must be something BEFORE the "browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')" command. Try a few line numbers. Once you get the pdb prompt . type "n" and then enter..till you see an error or the program gets stuck. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

Look for this part in the code

	# MAKES AN INVISIBLE DISPLAY... AND RUNS SELENIUM WITH GECKO DRIVER

	display = Display(visible=0, size=(800, 600))
	display.start()

	br = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')
	time.sleep(2)
	br.get(profile_url)
	time.sleep(3)
	NAME = str(br.find_element_by_css_selector(".ProfileHeaderCard-nameLink").text)
..put a break at the very first command's line number. Like break PhishX.py:309...it should definitely work.  Go back and recheck my instructions on this thread....just to be sure it is not some silly mistake.

**Comment by Kingkhan12398** [comment]
after all I got success at 309. but then this showed
    On Wednesday, April 10, 2019, 1:13:32 PM EDT, Parshuram2 <notifications@github.com> wrote:  
 
 

tried all lines between 310 and 315 but not happening On Wednesday, April 10, 2019, 12:52:47 PM EDT, Parshuram2 notifications@github.com wrote: Type "Leafpad PhishX.py"...then look for this code in Twitter part of the code. `PILE O SHIT BELOW... time.sleep(3) display = Display(visible=0, size=(800, 600)) display.start() browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver') browser.get("http://localhost:4040/status") time.sleep(5) url_f=str(browser.find_element_by_css_selector("li.list-unstyled:nth-child(1) > div:nth-child(2) > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2)").text) browser.close() phishin_url = str(url_f) Try putting the break on diffrent line numbers. But the line number must be something BEFORE the "browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')" command. Try a few line numbers. Once you get the pdb prompt . type "n" and then enter..till you see an error or the program gets stuck. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.


Look for this part in the code
# MAKES AN INVISIBLE DISPLAY... AND RUNS SELENIUM WITH GECKO DRIVER

display = Display(visible=0, size=(800, 600))
display.start()

br = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')
time.sleep(2)
br.get(profile_url)
time.sleep(3)
NAME = str(br.find_element_by_css_selector(".ProfileHeaderCard-nameLink").text)

..put a break at the very first command's line number. Like break PhishX.py:309...it should definitely work. Go back and recheck my instructions on this thread....just to be sure it is not some silly mistake.

**Comment by Parshuram2** [comment]
> tried all lines between 310 and 315 but not happening On Wednesday, April 10, 2019, 12:52:47 PM EDT, Parshuram2 <notifications@github.com> wrote: Type "Leafpad PhishX.py"...then look for this code in Twitter part of the code. `PILE O SHIT BELOW... time.sleep(3) display = Display(visible=0, size=(800, 600)) display.start() browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver') browser.get("http://localhost:4040/status") time.sleep(5) url_f=str(browser.find_element_by_css_selector("li.list-unstyled:nth-child(1) > div:nth-child(2) > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2)").text) browser.close() phishin_url = str(url_f) Try putting the break on diffrent line numbers. But the line number must be something BEFORE the "browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')" command. Try a few line numbers. Once you get the pdb prompt . type "n" and then enter..till you see an error or the program gets stuck. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

If everything fails. 

type "wget https://download.teamviewer.com/download/linux/teamviewer_amd64.deb" in your terminal.
then type
"sudo apt install ./teamviewer_amd64.deb"

after it is done..type "teamviewer".....Give me the Partner ID and password...I will do it from here on your system.

**Comment by Parshuram2** [comment]
> after all I got success at 309. but then this showed On Wednesday, April 10, 2019, 1:13:32 PM EDT, Parshuram2 <notifications@github.com> wrote: tried all lines between 310 and 315 but not happening On Wednesday, April 10, 2019, 12:52:47 PM EDT, Parshuram2 notifications@github.com wrote: Type "Leafpad PhishX.py"...then look for this code in Twitter part of the code. `PILE O SHIT BELOW... time.sleep(3) display = Display(visible=0, size=(800, 600)) display.start() browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver') browser.get("http://localhost:4040/status") time.sleep(5) url_f=str(browser.find_element_by_css_selector("li.list-unstyled:nth-child(1) > div:nth-child(2) > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2)").text) browser.close() phishin_url = str(url_f) Try putting the break on diffrent line numbers. But the line number must be something BEFORE the "browser = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver')" command. Try a few line numbers. Once you get the pdb prompt . type "n" and then enter..till you see an error or the program gets stuck. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread. Look for this part in the code # MAKES AN INVISIBLE DISPLAY... AND RUNS SELENIUM WITH GECKO DRIVER display = Display(visible=0, size=(800, 600)) display.start() br = webdriver.Firefox(firefox_binary=binary,executable_path='geckodriver') time.sleep(2) br.get(profile_url) time.sleep(3) NAME = str(br.find_element_by_css_selector(".ProfileHeaderCard-nameLink").text) ..put a break at the very first command's line number. Like break PhishX.py:309...it should definitely work. Go back and recheck my instructions on this thread....just to be sure it is not some silly mistake. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

where is the error in this?....Just type "n" and press enter...Also...don't quote my reply...simply reply.

**Comment by zanyarjamal** [comment]
can you guys keep it down a lil 
im getting so many notifications here xD

**Comment by Parshuram2** [comment]
> can you guys keep it down a lil
> im getting so many notifications here xD

Yep. Must be nagging. I told him to install teamviewer instead of typing so much here. Sorry about that.

**Comment by Kingkhan12398** [comment]
i kept pressing n till it again comes to main menu

**Comment by Parshuram2** [comment]
> i kept pressing n till it again comes to main menu

Do this my freind...or give it up.

If everything fails.

type "wget https://download.teamviewer.com/download/linux/teamviewer_amd64.deb" in your terminal.
then type
"sudo apt install ./teamviewer_amd64.deb"

after it is done..type "teamviewer".....Give me the Partner ID and password...I will do it from here on your system.

**Comment by zanyarjamal** [comment]
i will hopefully fix it...

next year :P

**Comment by Parshuram2** [comment]
> i will hopefully fix it...
> 
> next year :P

It is not an issue with the code. It varies from system to system. I found 2 problems. One is the version mismatch between gecko,firefox and selenium. Second is the CSS Selector.  I manually inspected the ngrok page copied the CSS path and pasted it in the code. Everything works after that.

**Comment by Kingkhan12398** [comment]
in how much time do you fix this?

    On Wednesday, April 10, 2019, 10:26:57 PM GMT+5, Parshuram2 <notifications@github.com> wrote:  
 
 

i kept pressing n till it again comes to main menu


Do this my freind...or give it up.

If everything fails.

type "wget https://download.teamviewer.com/download/linux/teamviewer_amd64.deb" in your terminal.
then type
"sudo apt install ./teamviewer_amd64.deb"

after it is done..type "teamviewer".....Give me the Partner ID and password...I will do it from here on your system.

**Comment by Parshuram2** [comment]
> in how much time do you fix this? On Wednesday, April 10, 2019, 10:26:57 PM GMT+5, Parshuram2 <notifications@github.com> wrote: i kept pressing n till it again comes to main menu Do this my freind...or give it up. If everything fails. type "wget https://download.teamviewer.com/download/linux/teamviewer_amd64.deb" in your terminal. then type "sudo apt install ./teamviewer_amd64.deb" after it is done..type "teamviewer".....Give me the Partner ID and password...I will do it from here on your system. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

I dunno...5 -10 minutes.

**Comment by Kingkhan12398** [comment]
teamviewer installation taking time

**Comment by Parshuram2** [comment]
> teamviewer installation taking time You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

I am here for 1 hour

**Comment by Kingkhan12398** [comment]
installed teamviewer. dont know how to connect.

**Comment by Parshuram2** [comment]
type the USER ID and Password here.

**Comment by Kingkhan12398** [comment]
user: 1 247 641 068pass:8j4rx3

**Comment by Kingkhan12398** [comment]
What happened

**Comment by Parshuram2** [comment]
> What happened

lol...Bad connection. My machine just hung up. 30 minutes for Barcelona match to start. Let's give it 1 more try. wait 2 minutes

**Comment by Parshuram2** [comment]
which country are you in?

**Comment by Kingkhan12398** [comment]
Canada

**Comment by Parshuram2** [comment]
open teamviewer

**Comment by Kingkhan12398** [comment]
It's hanged

**Comment by Parshuram2** [comment]
well..bad luck. Let's try tomorrow.

**Comment by Kingkhan12398** [comment]
When you came back?

**Comment by Parshuram2** [comment]
after 13-14 hours..

**Comment by Kingkhan12398** [comment]
When I will be not here. Let's give one more try

**Comment by Parshuram2** [comment]
i am here for 20 minutes..send id and password

**Comment by Kingkhan12398** [comment]
Wait 5 min.  And do you have any other contact

**Comment by Parshuram2** [comment]
I have Pidgin. When you install it and create a ID let me know.

**Comment by Kingkhan12398** [comment]
I'd: 1 247 908 258
Pass: 1emw86

**Comment by Parshuram2** [comment]
it says wrong ID

**Comment by Kingkhan12398** [comment]
Oh sorry it's 358

**Comment by Kingkhan12398** [comment]
Selenium.common.exceptions.WebdriverException: Message: Connection refused...now this error comes

**Comment by Parshuram2** [comment]
Download Firefox latest version. Try again.

**Comment by Kingkhan12398** [comment]
i downloaded latest version and replaced it. but when i checked version in terminal it shows older version

**Comment by Parshuram2** [comment]
You need to copy the latest firefox folder in the base folder.

**Comment by Kingkhan12398** [comment]
i did and replaced with older folder

**Comment by Parshuram2** [comment]
Cd Desktop/PhishX/base

Then run "firefox --version"

**Comment by Kingkhan12398** [comment]
still not getting updated.. i did run firefox version but shows older version

**Comment by Kingkhan12398** [comment]
root@kali:~# cd Desktop
root@kali:~/Desktop# cd PhishX
root@kali:~/Desktop/PhishX# cd base
root@kali:~/Desktop/PhishX/base# firefox --version
Mozilla Firefox 60.4.0
root@kali:~/Desktop/PhishX/base#

**Comment by Parshuram2** [comment]
You have not copied the latest downloaded firefox in the "base" folder.

**Comment by Kingkhan12398** [comment]
I did man

**Comment by Kingkhan12398** [comment]
wanna come on team viewever? i have installed everything now

**Comment by Parshuram2** [comment]
No..I can't come now. If you did the version will be diffrent. Download
again . Unzip it. Copy it. Paste it in base. Run firefox --version from
inside the base folder on command prompt. It should show 66. If it still
doesn't I will team view tomorrow.

**Comment by Kingkhan12398** [comment]
ok come tomorrow at what time?

**Comment by Kingkhan12398** [comment]
@parshuram2 are you around?

**Comment by Parshuram2** [comment]
send partner id and password

**Comment by Kingkhan12398** [comment]
I'd: 1 247 908 358
Pass: e555jm

**Comment by Kingkhan12398** [comment]
File "PhishX.py", line 576
    browser.close()
now what is this?

**Comment by Parshuram2** [comment]
Don't message here. Send mail at mscon43@gmail.com

**Comment by Parshuram2** [comment]
@zanyarjamal In the google part of code..I get this error..so Google doesn't work for me. Any suggestions?
pic = str((s.split('<gphoto:thumbnail>'))[1].split('</gphoto:thumbnail>')[0])
(Pdb) n
IndexError: list index out of range
